packages/vue/index.js

```diff
@@ -1,7 +1,5 @@
 'use strict'
 
-if (process.env.NODE_ENV === 'production') {
-  module.exports = require('./dist/vue.cjs.prod.js')
-} else {
-  module.exports = require('./dist/vue.cjs.js')
-}
+module.exports = process.env.NODE_ENV === 'production' ? 
+  require('./dist/vue.cjs.prod.js') : 
+  require('./dist/vue.cjs.js')
```

Comment:
?)
```
module.exports = require( process.env.NODE_ENV === 'production' ? './dist/vue.cjs.prod.js' : './dist/vue.cjs.js')
```

Reply:
Your current change is more readable. Thanks.